pkg/cmd/roachtest/test_runner.go

```diff
@@ -2032,6 +2032,11 @@ func (r *testRunner) maybeInitSideEyeClient(ctx context.Context, l *logger.Logge
 			"you can run the agent manually.")
 		return ""
 	}
+	if roachtestflags.Cloud == spec.IBM {
```

Comment:
Can remove entirely since side-eye is no more as of [1].

[1] https://github.com/cockroachdb/cockroach/pull/144982